test/es-module/test-esm-imports.mjs

```diff
@@ -26,6 +26,10 @@ const { requireImport, importImport } = importer;
     ['#subpath//asdf.asdf', { default: 'test' }],
     // Double slash
     ['#subpath/as//df.asdf', { default: 'test' }],
+    // Target steps below the package base
+    ['#belowbase', { default: 'belowbase' }],
```

Comment:
I know this is what it was called previously, but why is this “below” the base and not above the base?

Reply:
unclear but parent / below in the tree (towards the root) seems to make sense to me